src/client/index.html

```diff
@@ -42,6 +42,30 @@
       <!-- Hero Image goes here -->
 
       <!-- Firefox Download section goes here -->
+      <section id="download-section">
+          <div class="container">
+                <div class="row">
+                    <div class="col-md-8 col-sm-12 container">
```

Comment:
I think this is required for responsive design. Is there some other way to do this ?